### PR TITLE
Mapped and generated fake data

### PR DESCRIPTION
**Pull request by Sanjiiv-30**

### CODE REVIEW

File: data_type_match.py [hunk 1]
```diff
@@ -1,5 +1,95 @@
...
+            "FLOAT": lambda: self.pyfloat(min_value=0, max_value=214748.3647),
+            "INTEGER": lambda: self.pyint(0, 2147483647),
+            "INT": lambda: self.pyint(0, 2147483647),
+            "JSON": lambda: self.json(),
+            "LONGBLOB": lambda: self.binary(),
+            "LONGTEXT": lambda: self.sentence(),
+            "MEDIUMBLOB": lambda: self.binary(),
+            "MEDIUMINT": lambda: self.pyint(0, 8388607),
+            "MEDIUMTEXT": lambda: self.text(16777215),
+            "NCHAR": lambda: self.name(),
```
Comment (philosopherstonerush) [inline data_type_match.py:29]: We have not taken into account the different size constraint right? and also checks, defaults etc

File: data_type_match.py [hunk 2]
```diff
@@ -1,5 +1,95 @@
 from constants import MySQLDataType, PostgreSqlDataType, OracleDataType, MSSQLDataType
+from faker import Faker
 
+class gen_data(Faker):
```
Comment (philosopherstonerush) [inline data_type_match.py:4]: Really good use of class here, didnt think of something like this

File: data_type_match.py [hunk 3]
```diff
@@ -1,5 +1,95 @@
...
+            "REGCLASS": lambda: self.name(),
+            "REGCONFIG": lambda: self.language_name(),
+            "DOUBLE_PRECISION": lambda: self.pyfloat(right_digits=15),
+            "BYTEA": lambda: self.binary(),
+            "INTERVAL": lambda: self.time(),
+            "ARRAY": lambda: self.texts(),
+            "HSTORE": lambda: self.json_bytes(),
+            "INT4RANGE": lambda: f"[{self.pyint(1,100)},{self.pyint(101,200)}]",
+            "INT8RANGE": lambda: f"[{self.pyint(1,1000)},{self.pyint(1001,2000)}]",
+            "NUMRANGE": lambda: f"[{self.pyint(1.0,100.0)},{self.pyint(100.0,200.0)}]",
```
Comment (philosopherstonerush) [inline data_type_match.py:61]: I believe this needs a lot more of refactoring to be done to be maintainable haha

File: data_type_match.py (hunk 1 above)
Comment (Sanjiiv-30) [reply data_type_match.py:29]: Yes bro...It doesn't check any constraints...On a basic scale I have sliced the returned output to give the characters required by the user

File: data_type_match.py (hunk 2 above)
Comment (Sanjiiv-30) [reply data_type_match.py:4]: 🫂

File: data_type_match.py (hunk 3 above)
Comment (Sanjiiv-30) [reply data_type_match.py:61]: May be, but I guess it is not that vague

**Pull request merged**